RecoLocalMuon/RPCRecHit/src/CSCSegtoRPC.cc

```diff
@@ -127,48 +159,48 @@ CSCSegtoRPC::CSCSegtoRPC(edm::Handle<CSCSegmentCollection> allCSCSegments, const
 	  float dy=segmentDirection.y();
 	  float dz=segmentDirection.z();
 
-	  if(debug)  std::cout<<"Calling to Object Map class"<<std::endl;
-	  ObjectMapCSC* TheObjectCSC = ObjectMapCSC::GetInstance(iSetup);
-	  if(debug) std::cout<<"Creating the CSCIndex"<<std::endl;
-	  CSCStationIndex theindex(rpcRegion,rpcStation,rpcRing,rpcSegment);
-	  if(debug) std::cout<<"Getting the Rolls for the given index"<<std::endl;
+	  LogDebug("RPCPointProducer") << "Calling to Object Map class"<<std::endl;
+	  //ObjectMapCSC* TheObjectCSC = new ObjectMapCSC(iSetup);
```

Comment:
The debug message seems to refer to a commented-out line.  Both lines should probably be deleted. If you leave the commented-out line, please include a comment explaining why you are keeping it.